app/controller/user.go

```diff
@@ -125,6 +126,9 @@ func UpdateMe(c echo.Context) error {
 	}
 	user.Username = req.Username
 	user.Nickname = req.Nickname
+	if user.Email != req.Email {
+		user.EmailVerified = false
+	}
```

Comment:
Also set `emailVerified` in admin-user APIs?
https://github.com/EduOJ/backend/blob/34e752396fbbf094516f98adeac6f06730bba076/app/controller/admin_user.go#L17
https://github.com/EduOJ/backend/blob/34e752396fbbf094516f98adeac6f06730bba076/app/controller/admin_user.go#L51

Just mention, not important in fact, since that group of APIs are not usually used :-D

Reply:
In my opinion,we should allow admin changing a user's verified status

Reply:
@Jingyf-16

Reply:
I agree with leo, add the feild `emailVerified` to the request and set it if convenient